dockhand-lru-registry/values.yaml

```diff
@@ -0,0 +1,89 @@
+registry:
+  image:
+      repository: registry
+      pullPolicy: IfNotPresent
+      tag: 2.7.1
+  conf: |
+    version: 0.1
+    log:
+      fields:
+        service: registry
+    storage:
+      delete:
+        enabled: true
+      cache:
+        blobdescriptor: inmemory
+      filesystem:
+        rootdirectory: /var/lib/registry
+    http:
+      addr: :5000
+      headers:
+        X-Content-Type-Options: [nosniff]
+    health:
+      storagedriver:
+        enabled: true
+        interval: 10s
+        threshold: 3
+  cache:
+    enabled: true
+    emptyDir:
+      sizeLimitSoft: 10Gi
+      sizeLimitHard: 20Gi
```

Comment:
`sizeLimitHard` could be renamed `sizeLimit` but we might also want to make this optional